Lesson-7/lesson-7-new/style.css

```diff
@@ -0,0 +1,954 @@
+color: #FFFFFF; 
+}
+input[type="submit"]::placeholder { 
+font-family: Roboto;
+padding: 20px 57px;
+font-size: 24px;
+line-height: 28px;
+text-align: center;
+color: #FFFFFF; 
+} 
+input[type="text"] {
+width: 314px;
+font-family: Roboto;
+font-style: normal;
+font-weight: normal;
+font-size: 18px;
+line-height: 21px;
+text-align: center;
+color: #383838;
+}
+input[type="date"] {
+font-family: Roboto;
+font-style: normal;
+font-weight: normal;
+font-size: 18px;
+line-height: 21px;
+text-align: center;
+}
+input[type="search"]
+{
+width: 458px;
+font-family: Roboto;
+font-style: normal;
+font-weight: normal;
+font-size: 18px;
+line-height: 21px;
+color: #383838;
+}
+.phone_app {
+display: flex;
+justify-content: center;
+margin-top: 120px;
+width: 1232px;
+}
+.icon_google {
+display: block;
+width: 135px;
+height: 40px;
+}
+.icon_app_store {
+width: 117px;
+height: 38px;
+}
+.headline {
+font-family: Roboto;
+font-style: normal;
+font-weight: 500;
+font-size: 40px;
+line-height: 47px;
+width: 306px;
+height: 47px;
+margin-top: 120px;
+margin-left: 567px;
+}
+hr {
+margin: 0px 670px 88px 670px; 
+width: 100px;
+height: 4px;
+background: #F5BD41;
+}
+.container_headphones {
+display: flex;
+justify-content: center;
+flex-direction: column;
+width: 296px;
+height: 176px;
+}
+.container_communicate {
+display: flex;
+justify-content: center;
+flex-direction: column;
+width: 296px;
+height: 178px;
+}
+.container_gadgets {
+display: flex;
+justify-content: center;
+flex-direction: column;
+width: 296px;
+height: 184px;
+}
+.container_like {
+display: flex;
+justify-content: center;
+flex-direction: column;
+width: 296px;
+height: 183px;
+}
+.headphones {
+width: 75px;
+Height: 92px;
+margin-left: 110px;
+margin-bottom: 52px;
+}
+.communicate { 
+width: 83px;
+Height: 83px;
+margin-left: 105px;
+margin-bottom: 56px;
+}
+.gadgets {
+width: 100px;
+Height: 67px;
+margin-left: 98px;
+margin-bottom: 64px;
+}
+.like_img {
+width: 92px;
+Height: 83px;
+margin-left: 102px;
+margin-bottom: 60px;
+}
+.text_headphones {
+height: 28px;
+left: 77px;
+top: 148px;
+font-family: Roboto;
+font-style: normal;
+font-weight: normal;
+font-size: 24px;
+line-height: 28px;
+text-align: center;
+color: #383838;
+}
+.text_communicate {
+width: 296px;
+height: 28px;
+top: 148px;
+font-family: Roboto;
+font-style: normal;
+font-weight: normal;
+font-size: 24px;
+line-height: 28px;
+text-align: center;
+color: #383838;
+}
+.text_gadgets {
+height: 28px;
+left: 64px;
+top: 148px;
+font-family: Roboto;
+font-style: normal;
+font-weight: normal;
+font-size: 24px;
+line-height: 28px;
+text-align: center;
+color: #383838;
+}
+.text_like {
+width: 225px;
+height: 28px;
+left: 36px;
+top: 148px;
+font-family: Roboto;
+font-style: normal;
+font-weight: normal;
+font-size: 24px;
+line-height: 28px;
+text-align: center;
+color: #383838;
+}
+.all_containers {
+display: flex;
+justify-content: space-between;
+margin-left: 104px;
+margin-right: 103px;
+margin-bottom: 120px;
+}
```

Comment:
add max-width for this container 
![image](https://user-images.githubusercontent.com/20189313/128645847-201a2f6a-4271-4a85-88b0-775c36ce0450.png)